### PR TITLE
Resolve #1153 -- Fixed SuperMinion Spawn

### LINKED ISSUE #1153: Super Minions spawning on the wrong team
Destroying an inhibitor causes Super Minions to spawn on the side that had their inhibitor destroyed.

### PR DESCRIPTION
**Pull request by cabeca1143**
*Introduced GetOpposingTeamID
*Fixed Super Minions spawning on the wrong team

Resolve #1153

### CODE REVIEW

File: GameServerLib/Maps/SummonersRift.cs [hunk 1]
```diff
@@ -803,10 +803,10 @@ public bool Spawn()
                 }
 
                 List<Vector2> waypoint = new List<Vector2>();
-                TeamId team = barrack.Value.GetTeamID();
+                TeamId team = barrack.Value.GetOpposingTeamID();
```
Comment (danil179) [inline GameServerLib/Maps/SummonersRift.cs]: Change this variable to `oppose_team` as it can be confusing.

**Review by danil179 — COMMENTED** [review]
Confusing names of variables.

File: GameServerLib/Maps/SummonersRift.cs (hunk 1 above)
Comment (cabeca1143) [reply GameServerLib/Maps/SummonersRift.cs]: Resolved

**Review by danil179 — APPROVED** [review]
Approved.

**Pull request merged**